### ISSUE TITLE
implement zod type for google fonts

### ISSUE THREAD

**Issue by alexfernandez803** [remotion-dev/remotion#2692]
# Feature Request 🛍️
It would be beneficial to have a combobox or picker to select google fonts similar to zColor. This will improve the dev experience to easily select a font while development.
## Possible Solution
Extend https://www.remotion.dev/docs/zod-types/ to add a z-google-font with a zGoogleFont type

**Comment by JonnyBurger** [comment]
Sounds good to me, since google fonts is a huge package though, it would be better if it does not become a dependency of `@remotion/zod-types`. 

So maybe we only copy a list of the fonts as a string array into the zod-types package?

**Comment by alexfernandez803** [comment]
Yep, we could just create a script to generate an array from getAvailableFonts() tho write a file containing the font array.

Does the studio support combobox right now?

**Comment by JonnyBurger** [comment]
The studio does do that yes! It is just `z.enum`

**Comment by erayerdin** [comment]
Can I take this on? This will be my first time contributing to this project. :)

**Comment by JonnyBurger** [comment]
@erayerdin That would be awesome!
Thanks for giving it an attempt! 🙌

**Comment by JonnyBurger** [comment]
Not much movement on this issue, and I have a concern that this would 100x the bundle size of `@remotion/zod-types`.  
So I would say let's not leave it open as something we want to implement very soon.
